packages/ui/VirtualList/tests/VirtualList-specs.js

```diff
@@ -12,29 +41,34 @@ describe('VirtualList', () => {
 		handlerOnScrollStart,
 		handlerOnScrollStop,
 		items,
+		itemSize,
 		myScrollTo,
 		onScrollCount,
 		onScrollStartCount,
 		onScrollStopCount,
 		renderItem,
-		resultScrollTop;
+		resultScrollTop,
+		startScrollTop;
 
 	beforeEach(() => {
 		clientSize = {clientWidth: 1280, clientHeight: 720};
 		dataSize = 100;
 		items = [];
+		itemSize = 30;
 		onScrollCount = 0;
 		onScrollStartCount = 0;
 		onScrollStopCount = 0;
 		resultScrollTop = 0;
+		startScrollTop = 0;
 
 		getScrollTo = (scrollTo) => {
 			myScrollTo = scrollTo;
 		};
 		handlerOnScroll = () => {
 			onScrollCount++;
 		};
-		handlerOnScrollStart = () => {
+		handlerOnScrollStart = (e) => {
+			resultScrollTop = e.scrollTop;
```

Comment:
Is there a specific reason for this line?

Reply:
Oh, I intent "startScrollTop = e.scrollTop;".
It was mistake. sorry.